microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -157,9 +280,31 @@ To use the **Network Console**, complete the following steps.
 This experiment is available starting with Microsoft Edge version 85.
 
 
+<!-- ====================================================================== -->
+<!-- ## Focus Mode -->
+<!-- present in 96, 98 -->
+
+<!-- todo: add section content, link to it from What's New 96 and maybe from What's New 90 -->
```

Comment:
Need content for Focus Mode section of this Experimental Features article.

Reply:
Could we leave it as a quick one-sentence description (maybe from What's New 96) and create a new issue assigned to me to write this at a later time?

Reply:
Strange GitHub PR UI: the "Resolve conversation" button is in "Files changed" tab but not in the "Conversation" tab.

Added serviceable content here, will push.